platforms/db2/src/main/java/io/ebean/platform/db2/DB2Luw9Platform.java

```diff
@@ -0,0 +1,15 @@
+package io.ebean.platform.db2;
+
+import io.ebean.annotation.Platform;
+
+/**
+ * DB2 platform for Linux/Unix/Windows version 9.x.
+ */
+public class DB2Luw9Platform extends BaseDB2Platform {
```

Comment:
Note: There is still the `DB2LegacyPlatform` as fallback which has very restrictive limits to 18 characters. (For V7 and below).
I think it makes sense to include the limiters also there.